### ISSUE TITLE
Icon Request: fa-tableau

### ISSUE THREAD

**Issue by theclue** [FortAwesome/Font-Awesome#8942]
I think if it would be possible to add an icon for the popular visual analytics software [Tableau ](http://www.tableau.com/)and their very iconic logo (ok, very iconic to data analysts only, I admit)

![logo_tableau](https://cloud.githubusercontent.com/assets/1724406/14420001/cfc72600-ffc9-11e5-8743-9b94ce8af254.png)

This because media and analytics agencies very often use [their public gallery](https://public.tableau.com/s/gallery) as a social network, so the icon would be very oftern used in a corporate social icon list profile.

all the best and tnx for this wonderful opportunity,
gabriele

**Comment by mbeveridge** [comment]
+1 ('Tableau Public')

**Comment by DiscipleGeek** [comment]
![image](https://user-images.githubusercontent.com/12897305/33572064-797464ee-d8f7-11e7-9173-58cbf7468d1c.png)
This is the jpg of the one we use here at work. Would love an ico version.

**Comment by gazcn007** [comment]
+1

**Comment by Melissa13Medina** [comment]
+1 ('Tableau Public')

**Comment by gazcn007** [comment]
FYI, I can provide SVG and icon resources for both Tableau, and Tableau Public :)

**Comment by Melissa13Medina** [comment]
I'd be interested thanks, got a link?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by gazcn007** [comment]
Hi Melissa,
[link  for tableau logo](https://surveymonkey-assets.s3.amazonaws.com/papiasset/apps/logos/2e989404-aed0-41ea-9198-ddc1c76d7a4a)

[link for tableau public logo](https://public.tableau.com/v_201911902072005/images/public-logo-grey.png)

**Comment by anthonylei** [comment]
+1

**Comment by ridak** [comment]
+1

**Comment by hhnnhh** [comment]
+1

**Comment by dA505819** [comment]
Is the icon ready?

**Comment by DiscipleGeek** [comment]
I'm curious as to how long it takes to get an icon added to FA... I would think that this would be added already.

**Comment by angelabaltes** [comment]
I am also interested in this icon. Has it been added?

**Comment by vidhirathod** [comment]
true that , tableau icon will be good in fontawesome

**Comment by shambhavisingh24** [comment]
has the icon been added?

**Comment by nickgyoung** [comment]
+1

**Comment by bnithya15** [comment]
+1

**Comment by gazcn007** [comment]
FYI, the icon should be added as `fab-tableau` now:

https://fontawesome.com/icons?d=gallery&s=brands

**Comment by angelabaltes** [comment]
> FYI, the icon should be added as `fab-tableau` now:
> 
> https://fontawesome.com/icons?d=gallery&s=brands

I've just checked and I do not see the icon.

**Comment by tagliala** [comment]
> I've just checked and I do not see the icon.

Hi, the icon has not been implemented, this is a request

> FYI, the icon should be added as fab-tableau now:

I think it is about the name of this request

The current name is fine. Also, the correct syntax would be `fab fa-tableau`, so fa-tableau is fine (as well as tebleau)

**Comment by gazcn007** [comment]
> > I've just checked and I do not see the icon.
> 
> Hi, the icon has not been implemented, this is a request
> 
> > FYI, the icon should be added as fab-tableau now:
> 
> I think it is about the name of this request
> 
> The current name is fine. Also, the correct syntax would be `fab fa-tableau`, so fa-tableau is fine (as well as tebleau)

Yup I was referring to the name of the request. Maybe when we add it, it should be `fab fa-tableau` regarding the new brand naming convention.

**Comment by angelabaltes** [comment]
> > > I've just checked and I do not see the icon.
> > 
> > 
> > Hi, the icon has not been implemented, this is a request
> > > FYI, the icon should be added as fab-tableau now:
> > 
> > 
> > I think it is about the name of this request
> > The current name is fine. Also, the correct syntax would be `fab fa-tableau`, so fa-tableau is fine (as well as tebleau)
> 
> Yup I was referring to the name of the request. Maybe when we add it, it should be `fab fa-tableau` regarding the new brand naming convention.

Thank you for the clarification.

**Comment by lillar** [comment]
+1 for the Tableau icon it would be great to have it. Kudos for the dev team tho, fa is an amazing kit for amateurs like me:)

**Comment by yidaw123** [comment]
It's been a couple years, is the Tableau icon still a possibility?

**Comment by rohan1110** [comment]
here's one i found -
`<span class="iconify" data-icon="logos:tableau-icon" data-inline="false"></span>`

**Comment by SofiaKung** [comment]
+1

**Comment by kenirwin** [comment]
+1

**Comment by anhhchu** [comment]
+1

**Comment by Asikpalysik** [comment]
+1

**Comment by Asikpalysik** [comment]
Hello, I looking for fa-tableau so long can't find anything

**Comment by kolemae** [comment]
+1

**Comment by RakenduR** [comment]
+1

**Comment by stephperillo** [comment]
+1

**Comment by qnhtrn** [comment]
+1

**Comment by natvalenz** [comment]
please add Tableau

**Comment by git-larasati** [comment]
+1

**Comment by alicino** [comment]
Please, we need Tableau logo! +1

**Comment by puakehaulani** [comment]
+1

**Comment by jtw0mey** [comment]
+1

**Comment by IoanaMRusu** [comment]
+1

**Comment by Datapharm1** [comment]
Tableau logo, please. Thank you +1

**Comment by karthick12195** [comment]
+1

**Comment by theclue** [comment]
8 years and 77 reactions later, the icon is still missing unfortunately :| perhaps we should close the request at this stage...

**Comment by tsbarr** [comment]
+1 please, we need this icon!

**Comment by infiniteBI** [comment]
any update on the tableau public icon??

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
